q2_demux/_summarize/_visualizer.py

```diff
@@ -95,6 +95,17 @@ def _compute_stats_of_df(df):
     return df_stats
 
 
+def _build_seq_len_table(qscores: pd.DataFrame) -> str:
```

Comment:
Can we make these counts integers?

![screen shot 2018-06-19 at 7 05 50 am](https://user-images.githubusercontent.com/274668/41602250-3ac9acc6-738f-11e8-9bd7-cb1fe861d20e.png)

Maybe we could also tack on some units, too?

![screen shot 2018-06-19 at 7 08 32 am](https://user-images.githubusercontent.com/274668/41602422-9b09fcf8-738f-11e8-8b47-bbee894dbf6e.png)